editor/run/game_view_plugin.cpp

```diff
@@ -1070,6 +1070,9 @@ void GameView::_update_arguments_for_instance(int p_idx, List<String> &r_argumen
 
 void GameView::_window_close_request() {
 	if (window_wrapper->get_window_enabled()) {
+		// stop embedded process timer before close window wrapper,
+		// prevent send signal to focus EDITOR_GAME when window not enabled
```

Comment:
```suggestion
		// Stop embedded process timer before closing window wrapper,
		// to prevent sending signal to focus EDITOR_GAME when window is not enabled.
```